pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableBuilder.java

```diff
@@ -91,11 +94,16 @@
   private ByteBuffer _currentRowDataByteBuffer;
 
   public DataTableBuilder(DataSchema dataSchema) {
+    _version = VERSION_3;
     _dataSchema = dataSchema;
     _columnOffsets = new int[dataSchema.size()];
     _rowSizeInBytes = DataTableUtils.computeColumnOffsets(dataSchema, _columnOffsets);
   }
 
+  public static void setCurrentDataTableVersion(int version) {
+    _version = version;
```

Comment:
Throw exception if it is not one of the supported versions

Reply:
done